### PR TITLE
Sprint 1/issue 6/delete events option

### PR DESCRIPTION
**Pull request by AnaCarrizales**
"Eliminar evento" button was created as well as event partial

Screenshot from 2024-01-30 17-30-07
Screenshot from 2024-01-30 17-30-21

### CODE REVIEW

File: app/controllers/events_controller.rb [hunk 1]
```diff
@@ -1,11 +1,13 @@
 class EventsController < ApplicationController
-  before_action :set_event, only: [:show, :edit, :update]
+  before_action :set_event, only: [:show, :edit, :update, :destroy]
 
    def index
     @eventos = Event.all
+    @evento_partial = 'event'
```
Comment (MauricioNC) [inline app/controllers/events_controller.rb]: Delete this line as we discuss

File: app/controllers/events_controller.rb [hunk 2]
```diff
@@ -1,11 +1,13 @@
...
   end
 
   def show
+    @evento_partial = 'event'
```
Comment (MauricioNC) [inline app/controllers/events_controller.rb]: Delete this line too

**Review by MauricioNC — CHANGES_REQUESTED** [review]

**Review by MauricioNC — APPROVED** [review]
LGTM!

**Review by juperameza — APPROVED** [review]
LGTM

**Pull request merged**